common/test/package.scala

```diff
@@ -65,6 +66,7 @@ trait TestSettings {
 trait ConfiguredTestSuite extends ConfiguredServer with ConfiguredBrowser with ExecutionContexts {
   this: ConfiguredTestSuite with org.scalatest.Suite =>
 
+  lazy val webClient = new WebClient(BrowserVersion.CHROME)
```

Comment:
I can take a look at calling the controller directly - I ended up using HtmlUnit partly because my initial attempts needed to execute javascript.

HtmlUnit is just simulating Chrome (which I believe does not much more than changes to user-agent and javascript execution), so this implementation is pretty close to WS.url I think.